sormas-backend/src/main/java/de/symeda/sormas/backend/common/PatchHelper.java

```diff
@@ -0,0 +1,67 @@
+package de.symeda.sormas.backend.common;
+
+import java.lang.reflect.Field;
+import java.util.Set;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import net.minidev.json.JSONObject;
+
+public class PatchHelper {
+
+	/**
+	 * This method updates the existingObject with the values received in the jsonObject
+	 * 
+	 * @param jsonObject
+	 *            - the jsonObject contains the values that are intended to be updated,
+	 *            representing a subset of the fields of the object that is intended to be updated
+	 * @param updatingObjectClass
+	 *            - the class of the object that is intended to be updated
+	 * @param existingObject
+	 *            - the existingObject that is intended to be updated
+	 * @param <T>
+	 * @return
+	 */
+	public static <T> T postUpdate(JSONObject jsonObject, Class<T> updatingObjectClass, T existingObject) {
```

Comment:
Please add a unit test for this. Especially one that passes in an hierarchical dto, like a case and one that covers a DTO with lists. We need to make sure that this does not throw away child objects

Reply:
I will add a unit test. I tested it and it does not throw away child objects.